Jenkinsfile

```diff
@@ -33,7 +33,6 @@ pipeline {
             when {
                 anyOf {
                     branch 'master'
-                    branch 'release-*'
```

Comment:
will this release branch filter (or something similar) need to be added back in later when we are ready to publish a real release?